src/main/java/org/openrewrite/java/spring/boot2/LoggingShutdownHooksRecipe.java

```diff
@@ -0,0 +1,74 @@
+package org.openrewrite.java.spring.boot2;
```

Comment:
Since this is just a search and mark recipe, can you move this to a new search package, `org.openrewrite.java.spring.boot2.search`.